tests/ferret/core_test.go

```diff
@@ -107,7 +107,7 @@ func TestCore(t *testing.T) {
 		// TODO keep in sync with test_db? https://github.com/FerretDB/dance/issues/43
 		data := map[string]any{
 			"double":                   42.13,
-			"double-zero":              0.0,
+			"double-zero":              0.0000001,
```

Comment:
I found on the forum that `0.0000001` value is used to insert the 0(zero) as a double data type

Reply:
If `double-smallest` is sufficient, I will remove the double zero.

Reply:
I think that is already covered by `double-smallest`

Reply:
OK. In this case, It's okay to remove the "double zero" field, because it inserts `0.0` into stores as an int and not as a double data type?

Reply:
It should be double (float64). I think we even have tests for that. If it is not the case, it is worth creating a separate issue and PR.

Reply:
Well, I just double-checked and made sure that through mongosh the `0.0` value is inserted into mongodb as int and it does. But since we insert values through go code, it sets a type to "double-zero" value as a `double`. So it turns out there is no particular problem in the double zero, I will leave the value as 0.0

Reply:
That sounds like mongosh's [JavaScript problem](https://www.destroyallsoftware.com/talks/wat) 😆